diff --git a/src/Commands/LegacyCommands.php b/src/Commands/LegacyCommands.php
index d5a7fcd6..0f2db839 100644
--- a/src/Commands/LegacyCommands.php
+++ b/src/Commands/LegacyCommands.php
@@ -125,18 +125,6 @@ class LegacyCommands extends DrushCommands
     {
     }

-    /**
-     * field-create has been deprecated. Please try `generate field` command.
-     *
-     * @command field:create
-     * @aliases field-create
-     * @hidden
-     * @obsolete
-     */
-    public function field()
-    {
-    }
-
     /**
      * core:execute has been deprecated. Please try `site:ssh` command.
      *
